lib/presentation/screens/details_screen/bloc/details_screen_state.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:marvelapp_flutter/data/model/response_models/series.dart';
+import '../../../../data/model/response_models/character.dart';
+
+@immutable
+class DetailsScreenState extends Equatable {
+  final bool loading;
+  final Character? character;
+  final String? error;
+  final List<Series>? series;
+
+  const DetailsScreenState({
+    required this.loading,
```

Comment:
Instead of required you can set default value to true like `this.loading = true,` and skip it when call `copyWith()` in case you no need loading.
The same other parameters, for them default value can be `null`

Reply:
i try , it don't worked. i try like this
 `const DetailsState({
    this.loading = true,
    this.character,
    this.error,
    this.series,
  });

  DetailsState copyWith({
    Character? character,
    String? error,
    List<Series>? series,
  }) {
    return DetailsState(
      character: character ?? this.character,
      series: series ?? this.series,
      error: error,
    );
  }`

Reply:
Indeed, in this case it does not make much sense.